### PR TITLE
Updated code and docs for sbs-gen1 and sbs-gen2

### PR DESCRIPTION
**Pull request by Sativarsainath-26**
Updated required document and code to work in sbs-gen1 and gen2 environment.  
Please review the code and document, let me know if anything is missing.

### CODE REVIEW

**Review by priyanayar — COMMENTED** [review]
I reviewed the 3 doc files and based on our discussions, I made the updates and raised this PR: https://github.ibm.com/ZaaS/secure-build-cli/pull/141

**Review by abhiramkulkarni — APPROVED** [review]

**Comment by Sativarsainath-26** [comment]
> I reviewed the 3 doc files and based on our discussions, I made the updates and raised this PR: https://github.ibm.com/ZaaS/secure-build-cli/pull/141

@priyanayar Updated the latest document, Please review it. let me know if any link is breaking.

File: client_certificate.py [hunk 1]
```diff
@@ -237,7 +247,7 @@ def exit_if_exist(self, *files):
                 existing_files.append(file)
         if len(existing_files) > 0:
             logging.error('client_certificate: exiting because the following files already exist - {}'.format(' '.join(existing_files)))
-            logging.error('client_certificate: specify CLIENT_CRT_KEY, CAPATH, and CAKEYPATH or empty the directory')
+            logging.error('client_certificate: speficy CLIENT_CRT_KEY, CAPATH, and CAKEYPATH or empty the directory')
```
Comment (Ankitha140) [inline client_certificate.py:250]: @Sativarsainath-26 typo error

**Review by Ankitha140 — APPROVED** [review]
LGTM

File: client_certificate.py (hunk 1 above)
Comment (Sativarsainath-26) [reply client_certificate.py]: Thanks @Ankitha140, updated the code https://github.com/ibm-hyper-protect/secure-build-cli/pull/34/commits/72396177a689856b0746facf11665b998b063f82 :slightly_smiling_face:

**Pull request merged**